packages/uikit-react-native/src/fragments/createGroupChannelFragment.tsx

```diff
@@ -197,7 +209,10 @@ const createGroupChannelFragment = (initModule?: Partial<GroupChannelModule>): G
             renderNewMessagesButton={renderNewMessagesButton}
             renderScrollToBottomButton={renderScrollToBottomButton}
             onResendFailedMessage={resendMessage}
-            onDeleteMessage={deleteMessage}
+            onDeleteMessage={async (message) => {
+              await deleteMessage(message);
+              await onAfterDeleteMessage(message);
+            }}
```

Comment:
Can we build to make sure there are no errors? I'm worried about potentially using a function with a different signature.